src/zui/ZUICreatePerson/PersonalInfoForm.tsx

```diff
@@ -0,0 +1,247 @@
+    >
+      <Box display="flex" mt={1}>
+        <Box mr={2} width="50%">
+          <InfoInputForm
+            field={'first_name'}
+            onChange={(field, value) => onChange(field, value)}
+            required
+          />
+        </Box>
+        <Box width="50%">
+          <InfoInputForm
+            field={'last_name'}
+            onChange={(field, value) => onChange(field, value)}
+            required
+          />
+        </Box>
+      </Box>
+      <InfoInputForm
+        error={invalidFields.includes('email')}
+        field={'email'}
+        onChange={(field, value) => onChange(field, value)}
+      />
+      <InfoInputForm
+        error={invalidFields.includes('phone')}
+        field={'phone'}
+        onChange={(field, value) => onChange(field, value)}
+      />
+      {showAllClickedType !== 'none' && (
+        <Box display="flex" flexDirection="column" gap={2}>
+          <InfoInputForm
+            error={invalidFields.includes('alt_phone')}
+            field={'alt_phone'}
+            inputRef={inputRef}
+            onChange={(field, value) => onChange(field, value)}
+          />
+          <FormControl fullWidth>
+            <InputLabel>
+              <Msg id={globalMessageIds.personFields.gender} />
+            </InputLabel>
+            <Select
+              defaultValue=""
+              label={globalMessages.personFields.gender()}
+              onChange={(e) =>
+                onChange(
+                  'gender',
+                  e.target.value === 'unknown' ? '' : e.target.value
+                )
+              }
+            >
+              {genderKeys.map((genderKey) => (
+                <MenuItem key={genderKey} value={genderKey}>
+                  {messages.createPerson.genders[genderKey]()}
+                </MenuItem>
+              ))}
+            </Select>
+          </FormControl>
+          <InfoInputForm
+            field={'street_address'}
+            onChange={(field, value) => onChange(field, value)}
+          />
+          <InfoInputForm
+            field={'co_address'}
+            onChange={(field, value) => onChange(field, value)}
+          />
+          <Box>
+            <InfoInputForm
+              field={'zip_code'}
+              onChange={(field, value) => onChange(field, value)}
+              style={{
+                pr: 2,
+                width: '30%',
+              }}
+            />
+            <InfoInputForm
+              field={'city'}
+              onChange={(field, value) => onChange(field, value)}
+              style={{
+                width: '70%',
+              }}
+            />
+          </Box>
+          <InfoInputForm
+            field={'country'}
+            onChange={(field, value) => onChange(field, value)}
+          />
+          <InfoInputForm
+            field={'ext_id'}
+            onChange={(field, value) => onChange(field, value)}
+          />
+        </Box>
+      )}
+      {showAllClickedType !== 'none' &&
```

Comment:
As an example, if `ShowAllTrigeredType` included `null` instead of `'none'`, this line of code could be simplified as:

```tsx
{ !!showAllClickedType &&
```